src/traits/impls.rs

```diff
@@ -46,6 +51,23 @@ impl DoughnutApi for () {
     }
 }
 
+#[cfg(feature = "std")]
+impl Signing for ParityDoughnutV0 {
+    fn sign_ed25519(&mut self, secret_key: &[u8]) -> Result<H512, SigningError> {
+        match sign_ed25519(&self.issuer(), secret_key, &self.payload()) {
+            Ok(signed_signature) => Ok(H512::from_slice(&signed_signature.as_slice())),
+            Err(e) => Err(e),
+        }
+    }
+
+    fn sign_sr25519(&mut self, secret_key: &[u8]) -> Result<H512, SigningError> {
+        match sign_sr25519(&self.issuer(), secret_key, &self.payload()) {
+            Ok(signed_signature) => Ok(H512::from_slice(&signed_signature.as_slice())),
```

Comment:
Can we change this to return `Vec<u8>` since it's a simpler type for clients (i.e. they won't need to import H512).
`H512` is an unfortunate type we use for compatibility in plug/substrate.

Reply:
It seems signature is defined as H512 in ParityDoughnutV0 - https://github.com/cennznet/doughnut-rs/blob/enable-signing/src/v0/parity.rs#L228

Reply:
@holygits Is there a type like `any`?